js/jquery.validation.js

```diff
@@ -43,7 +43,8 @@
                 alert: {
                     classes: 'message-area alert'
                 }
-            }
+            },
+	    onValidSubmit: true
```

Comment:
Bernhard had suggested that `submit` should be appropriate as a name and I'm inclined to agree now. It makes sense that we don't consider the validation part as a submission; only the submission to actually change the data.

Can this option be renamed to `submit` please?